### PR TITLE
Update secure transport

### PR DESCRIPTION
**Pull request by cammellos**
Update secure transport docs, remove non-implemented features.

### CODE REVIEW

File: status-account-spec.md [hunk 1]
```diff
@@ -156,10 +156,7 @@ This can be done in a the following ways:
   - include BundleContainer???
 - a new bundle SHOULD be created at least every 12 hours
 - a bundle is only generated when it is used
-- a bundle MUST be distributed on the contact code channel (NOTE: define this where?)
-
-#### QR code
-- A generated QR code should include a X3DH bundle set along with the contact code but I can't find the code to do so.
+- a bundle SHOULD be distributed on the contact code channel. This is the whisper topic `{IK}-contact-code`, where `IK` is the hex encoded public key of the user. The channel is encrypted in the same way public chats are encrypted.
```
Comment (adambabik) [inline status-account-spec.md]: Hex encoded with `0x` as a prefix or without?

Comment (cammellos) [reply status-account-spec.md]: good point , prefixed with `0x` , updated specs

File: status-secure-transport-spec.md [hunk 2]
```diff
@@ -106,9 +106,7 @@ Every client initially generates some key material which is stored locally:
 
 More details can be found in the `X3DH Prekey bundle creation` section of [Account specification](./status-account-spec.md#x3dh-prekey-bundle-creation).
 
-A `contact-code` is a protobuf `Bundle` message, encoded in `JSON` and converted to their `base64` string representation.
-
-Prekey bundles are can be extracted from any user's messages, or found via searching for their specific contact code topic, `{IK}-contact-code`.
+Prekey bundles are can be extracted from any user's messages, or found via searching for their specific topic, `{IK}-contact-code`.
```
Comment (corpetty) [inline status-secure-transport-spec.md]: `are can be` should be changed for proper grammar.

File: status-session-management-spec.md [hunk 3]
```diff
@@ -39,34 +39,35 @@ A new session is initialized once a successful X3DH exchange has taken place. Su
 
 ## Concurrent sessions
 
-If two sessions are created concurrently between two peers the one with the symmetric key, first in byte order should be used this marks that the other has expired.
+If two sessions are created concurrently between two peers the one with the symmetric key, first in byte order SHOULD be used this marks that the other has expired.
```
Comment (corpetty) [inline status-session-management-spec.md]: no `,` needed

Comment (corpetty) [reply status-session-management-spec.md]: add a `,` after `SHOULD be used`

**Review by corpetty — CHANGES_REQUESTED** [review]
minor changes, nitpicky but might as well get them done. Great updates.

**Comment by cammellos** [comment]
@adambabik @corpetty ready to be reviewed again, thanks!

**Review by corpetty — APPROVED** [review]

**Pull request merged**